src/main.jsx

```diff
@@ -2,13 +2,16 @@
 
 import "babel-core/polyfill";
 import "./lib/auth.js";
-import "!style!css!less!./main.less"
+import "!style!css!less!./main.less";
 
-import About from "./components/about.jsx";
+import Header from "./components/header";
```

Comment:
Let's keep `header.jsx` or remove `jsx` extension from the other imports for consistency.